src/Compilers/VisualBasic/Portable/Parser/ParseConditional.vb

```diff
@@ -460,6 +472,44 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return SyntaxFactory.ReferenceDirectiveTrivia(hashToken, referenceKeyword, file)
         End Function
 
+        Private Function ParseShebangDirective(hashToken As PunctuationSyntax) As ShebangDirectiveTriviaSyntax
+            Dim builder = New StringBuilder()
+            Debug.Assert(CurrentToken.Kind = SyntaxKind.ExclamationToken,
+                         NameOf(ParseShebangDirective) & " called with wrong token")
+            Dim exclamationToken = DirectCast(CurrentToken, PunctuationSyntax)
+            GetNextToken()
+
+            ' Eat tokens until the end of line.
+            While CurrentToken.Kind <> SyntaxKind.EndOfLineTrivia AndAlso
```

Comment:
Will `CurrentToken` ever be an `EndOfLineTrivia`?  That doesn't sound like a token.